javascript/templates/api/api.mustache

```diff
@@ -208,17 +260,16 @@ export class {{classname}}ResponseProcessor {
                 "{{{returnType}}}", "{{returnFormat}}"
             ) as {{{returnType}}};
             {{/isBinary}}
-            return body;
+            return new HttpInfo(response.httpStatusCode, response.headers, response.body, body);
             {{/returnType}}
             {{^returnType}}
-            return;
+            return new HttpInfo(response.httpStatusCode, response.headers, response.body, undefined);
             {{/returnType}}
         }
-
-        let body = await response.body.text();
-        throw new ApiException<string>(response.httpStatusCode, body);
+        // FIXME(onelson): verify that special handling of empty body is no longer required.
```

Comment:
old comment, right? you already have a test

Reply:
I think you're right. Thanks.